### PR TITLE
bedops: add mantainer

### PR DESCRIPTION
**Pull request by jacorvar**

### CODE REVIEW

File: var/spack/repos/builtin/packages/bedops/package.py [hunk 1]
```diff
@@ -15,6 +15,8 @@ class Bedops(MakefilePackage):
     homepage = "https://bedops.readthedocs.io"
     url      = "https://github.com/bedops/bedops/archive/v2.4.39.tar.gz"
 
+    maintainers = ['jacorvar']
+
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/bedops/package.py]: Style error is complaining about whitespace.  The following should fix it:

```suggestion

```

**Comment by tldahlgren** [comment]
@erimar77 @alecbcs As prior contributors to the package, do you also want to be added as maintainers?

**Review by tldahlgren — CHANGES_REQUESTED** [review]
See prior comment about the style error.

**Comment by tldahlgren** [comment]
@jacorvar Please resolve the file conflicts.

**Review by jacorvar — COMMENTED** [review]
I've just changed it.

File: var/spack/repos/builtin/packages/bedops/package.py [hunk 2]
```diff
@@ -15,7 +15,8 @@ class Bedops(MakefilePackage):
     homepage = "https://bedops.readthedocs.io"
     url      = "https://github.com/bedops/bedops/archive/v2.4.39.tar.gz"
 
-    version('2.4.40', sha256='8c01db76669dc58c595e2e1b9bdb6d462f3363fc569b15c460a63a63b8b6bf30')
+    maintainers = ['jacorvar']
+    version('2.4.40', sha256='0670f9ce2da4b68ab13f82c023c84509c7fce5aeb5df980c385fac76eabed4fb')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/bedops/package.py:19]: This is an invalid sha256 BUT #26141 has already been merged and it contains the maintainer and correct sha256 for this version.

**Comment by tldahlgren** [comment]
Closing since these changes appear in a PR (#26141) that has already been merged.

**Pull request closed without merging**